_posts/2024-08-16-logdetective-reviews.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Reviewing annotations in Log Detective"
+author: "Tomas Tomecek"
+layout: "post"
+---
+
+We appreciate everyone who submitted annotations, thank you! It will allow us to
+create a model with knowledge about building software, not just RPMs, but also
+container images, test failures and various language specific buildsystems. I
+envision that Log Detective will provide useful answers to most of logs with
+build failures. Though we still have a long way getting there.
+
+So far, the annotations on the [logdetective.com](https://logdetective.com/)
+site are read-only. Once submitted, there is no way to change them. Everyone
+can download them, but that's it.
+
+Starting in August 2024, we have now deployed a review interface on the Log
+Detective website. This will allow anyone to rate (thumbs up and down) each
+snippet and therefore curate collected data.
+
+![](/assets/img/posts/logdetective-with-review.png)
+
+Annotations combined with your rating will enable us create a high-quality data
+set and increase Log Detective's chance giving you solid advice.
+
+We've already done research on how to utilize this data inside Log Detective but
+haven't decided yet. Please stay tuned.
+
+![](/assets/img/posts/logdetective-review-submit.png)
+
+## How do I review snippets?
+
+Once you click the "Review logs" button at the top menu, you'll reach the new
+review interface. The instructions are on the left side.
+
+![](/assets/img/posts/logdetective-review-instructions.png)
+
+You are welcome to add more snippets if you can spot more information that
+should be annotated. If an annotation is incorrect, please downvote it and
+create a new one.
```

Comment:
I don't know if to mention it in this paragraph or a different one but if a snippet is correctly highlighted only the comment is incorrect, it is possible to edit the comment.

Reply:
oh, very good point!